lib/msf/core/post/common.rb

```diff
@@ -112,9 +112,15 @@ def cmd_exec(cmd, args=nil, time_out=15, opts = {})
       opts = {
         'Hidden' => true,
         'Channelized' => true,
-        'Subshell' => true
+        'Subshell' => true,
+        'CaptureOutput' => true
```

Comment:
Instead of creating a new `CaptureOutput` option, is it possible to just reuse the `Channelized` flag? 👀